### PR TITLE
Improved speeding sol

### PR DESCRIPTION
**Pull request by SansPapyrus683**
Added (and tested) Java + Python sol.

### CODE REVIEW

File: solutions/usaco-568.mdx [hunk 1]
```diff
@@ -13,49 +13,128 @@ author: Jesse Choe
...
+	limit_segs = [[int(i) for i in read.readline().split()]
+				   for _ in range(road_seg_num)]
+
+	bessie_segs = [[int(i) for i in read.readline().split()]
+					for _ in range(speed_seg_num)]
+
+limit = []
+for s in limit_segs:
+	for _ in range(s[0]):
+		limit.append(s[1])
```
Comment (maggie-j-liu) [inline solutions/usaco-568.mdx:127]: you could use extend here, but up to you

File: solutions/usaco-568.mdx [hunk 2]
```diff
@@ -13,49 +13,128 @@ author: Jesse Choe
...
+	cout << worst << endl;
 }
-
 ```
 
 </CPPSection>
+<JavaSection>
+
+```java
+import java.io.*;
```
Comment (maggie-j-liu) [inline solutions/usaco-568.mdx:62]: idt this is necessary since the official sol is java

Comment (SansPapyrus683) [reply solutions/usaco-568.mdx:62]: what's the harm in adding an alternative implementation? other than a minor increase in file size

Comment (maggie-j-liu) [reply solutions/usaco-568.mdx:62]: well it's the same idea as the offical sol, so it's repetitive

**Review by maggie-j-liu — APPROVED** [review]
> yeah seems good enough

**Pull request merged**